android/src/main/java/app/notifee/core/utility/ExtendedListenableFuture.java

```diff
@@ -0,0 +1,74 @@
+package app.notifee.core.utility;
+
+import com.google.common.util.concurrent.AsyncFunction;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.common.util.concurrent.Futures;
+import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
will those pass f-droid blocklist checks? I have not investigated but 'com.google.nnn' seems suspect vs 'androidx.nnn'. You may have already investigated?

Reply:
I haven't test it yet

Reply:
[it should be fine](https://gitlab.com/fdroid/fdroidserver/-/blob/master/fdroidserver/scanner.py)